sidebars.yaml

```diff
@@ -485,6 +492,7 @@
                     - page: docs/references/http-websocket-apis/public-api-methods/path-and-order-book-methods/nft_sell_offers.md
                     - page: docs/references/http-websocket-apis/public-api-methods/path-and-order-book-methods/path_find.md
                     - page: docs/references/http-websocket-apis/public-api-methods/path-and-order-book-methods/ripple_path_find.md
+                    - page: docs/references/http-websocket-apis/public-api-methods/path-and-order-book-methods/mpt_holders.md
```

Comment:
Since this method is only available in Clio, could be listed under Clio methods.

Reply:
Moved.